src/main/webapp/course.html

```diff
@@ -165,7 +149,7 @@ <h1 class="p-4 text-center">Add Your Review</h1>
           </div>
         </form>
         <button
-          onclick="passRatingProperties()"
+          id="button"
```

Comment:
I think I changed this to id = "form-submit" because Luke said to make it more specific so you will get a merge conflict regarding this I believe.

Reply:
Will change that as well!